src/yang/example-qos-configuration-a.1.5.xml

```diff
@@ -0,0 +1,162 @@
+          <type>qt:traffic-group-name</type>
+          <logical-not>false</logical-not>
+          <traffic-group
+    	      xmlns="urn:ietf:params:xml:ns:yang:ietf-queue-policy">
+            <name>video</name>
+          </traffic-group>
+        </filter>
+      </inline> 
+      <action>
+	<type>qt:queue</type>
+	<queue
+	    xmlns="urn:ietf:params:xml:ns:yang:ietf-queue-policy">
+	  <priority>
+	    <level>2</level>
+	  </priority>
+	  <min-rate>
+	    <value>10000000</value>
+	    <unit>qt:bits-per-second</unit>
+	  </min-rate>
+	  <max-rate>
+	    <value>30000000</value>
+	    <unit>qt:bits-per-second</unit>
+	  </max-rate>
+	</queue>
+      </action>
+    </classifier>
+    <classifier>
+      <name>DATA</name>
+      <inline> 
+        <filter>
+          <type>qt:traffic-group-name</type>
+          <logical-not>false</logical-not>
+          <traffic-group
+    	      xmlns="urn:ietf:params:xml:ns:yang:ietf-queue-policy">
+            <name>data</name>
+          </traffic-group>
+        </filter>
+      </inline> 
+      <action>
+	<type>qt:queue</type>
+	<queue
+	    xmlns="urn:ietf:params:xml:ns:yang:ietf-queue-policy">
+	  <priority>
+	    <level>2</level>
+	  </priority>
+	  <min-rate>
+	    <value>8000000</value>
+	    <unit>qt:bits-per-second</unit>
+	  </min-rate>
+	  <max-rate>
+	    <value>18000000</value>
+	    <unit>qt:bits-per-second</unit>
+	  </max-rate>
+	</queue>
+      </action>
+    </classifier>
+  </policy>
+  <policy>
+    <name>SCHEDULING-POLICY</name>
+    <type>qt:scheduler-policy-type</type>
+    <classifier>
+      <name>ALL</name>
+      <inline> 
+        <filter>
+          <type>qt:filter-match-all</type>
+          <logical-not>false</logical-not>
+          <sp:match-all>
+            <sp:action></sp:action>
+          </sp:match-all>
+        </filter>
+      </inline> 
+      <action>
+	<type>qt:scheduler</type>
+	<scheduler
+            xmlns="urn:ietf:params:xml:ns:yang:ietf-scheduler-policy">
+	  <min-rate>
+	    <value>20000000</value>
+	    <unit>qt:bits-per-second</unit>
+	  </min-rate>
+	  <max-rate>
+	    <value>50000000</value>
+	    <unit>qt:bits-per-second</unit>
+	  </max-rate>
+	</scheduler>
+      </action>
+      <action>
+	<type>qt:queue-policy-name</type>
+	<queue-policy-name
+            xmlns="urn:ietf:params:xml:ns:yang:ietf-scheduler-policy">
+          <queue-policy>QUEUEING-POLICY</queue-policy>
+	</queue-policy-name>
+      </action>
+    </classifier>
+  </policy>
+</policies>
+
+<interfaces
+    xmlns="urn:ietf:params:xml:ns:yang:ietf-interfaces"
+    xmlns:qt="urn:ietf:params:xml:ns:yang:iana-qos-types"
+    xmlns:ianaift="urn:ietf:params:xml:ns:yang:iana-if-type">
+  <interface>
+    <name>FastEthernet 0/0</name>
+    <type>ianaift:ethernetCsmacd</type>
+    <qos-target-policy
+        xmlns="urn:ietf:params:xml:ns:yang:ietf-traffic-policy">
+      <direction>qt:egress</direction>
+      <type>qt:scheduler-policy-type</type>
+      <name>SCHEDULING-POLICY</name>
```

Comment:
I thought the example was going to be a combination of queue and scheduler policy. If we are not using QUEUEING-POLICY, can we remove it? Or maybe add more lines here to use QUEUEING-POLICY??

Reply:
Ahh! I see.